src/libs/actions/BankAccounts.ts

```diff
@@ -113,7 +113,7 @@ function setPersonalBankAccountContinueKYCOnSuccess(onSuccessFallbackRoute: Rout
 
 function clearPersonalBankAccount() {
     clearPlaid();
-    Onyx.set(ONYXKEYS.PERSONAL_BANK_ACCOUNT, {});
+    Onyx.set(ONYXKEYS.PERSONAL_BANK_ACCOUNT, null);
```

Comment:
Changed this to `null` to fix the type. Every usage of `personalBankAccount` already uses a null safety, so it's safe.